src/app/components/challenge/challengesubmissions/challengesubmissions.component.html

```diff
@@ -1,67 +1,166 @@
-<div class="challenge-submissions-container">
-  <div class="select-phase-container">
-    <app-selectphase [phases]="filteredPhases" [phaseSelected]="phaseSelected()" #phaseselect></app-selectphase>
-    <div *ngIf="!selectedPhase" class="fw-bold">
-      Please select a phase.
+<div class="challenge-card">
+  <div class="ev-card-panel card-bt-margin">
+    <div class="ev-md-container bottom-hr-line">
```

Comment:
Formatting is a bit off in this file.
See my patch which has formatting from intellij.

[format_this_way.patch.zip](https://github.com/Cloud-CV/EvalAI-ngx/files/3389989/format_this_way.patch.zip)

Reply:
done